### DISCUSSION TITLE
New Extend Updates

### DISCUSSION THREAD

**Discussion by cdllc** [primefaces/community#2396] [PrimeReact Templates] [answered]
Updates were unrestricted when I purchased my licenses, including updates. Changing the terms of the license without my authorization to now charge me, aside from being illegal, will put PrimeFaces under scrutiny and ultimately cause others to distrust the organization.

Those who purchased licenses when there was no limit to updates should be grandfathered into receiving updates perpetually. You are legally liable for altering the terms of the license after the original conditions were accepted in a monetary transaction.

I understand the need to fund your initiatives to keep quality updates coming (I've noticed), but a precedent was established at the time of payment.

**Comment by melloware** [comment]
@cdllc can you clarify what you are talking about?   I wasn't aware of any changes to Template licensing?

**Comment by cagataycivici** [comment]
Hi, the Store will not charge anything without your consent, it is not renewed automatically, it is not a subscription, updates are opt-in. 

We've also granted 1 year+ free of charge to any user regardless of purchase date, even to those who purchased a template 5 years ago. The template can be used indefinitely for lifetime, only free updates are available for 1 year.

**Reply by cdllc** [reply, marked as answer]
Requiring payment for updates, when previously providing them without charge, is a change to the license terms by placing a restriction on what once was available free of charge. This was definitely part of my evaluation criteria to decide what to base my work around, without a doubt, others too.

Added restrictions thereafter constitutes change in license. 

> free updates are available for 1 year.

This is what has changed. This restriction did not exist prior, nor was a restriction like this enforced.
